### PR TITLE
This adds admin site features for managing database items

### PR DESCRIPTION
**Pull request by hamadhassan3**
# Overview of the Work
This project is part of the contents for 3rd week in the training program for python developers. It uses Django web framework for implementing a browse-only library.

# Major Changes
Practice on admin site features like adding, deleting and modifying data. Other features like inline editing, better representation of attributes and foreignkey attributes representation using ModelAdmin.

# Output
The admin site is displayed and data can be viewed or modified.
Capture
author_representation
author_details

# Feedback Required
I need feedback on the quality of code (comments, variable names, etc.) as well as the overall style and use of good practices.

### CODE REVIEW

File: catalog/admin.py [hunk 1]
```diff
@@ -1,3 +1,14 @@
...
+
+
+# The models are registered here so that they can be populated by admin site
+# All models that are registered here are accessible through admin site
+
+admin.site.register(Book)
+admin.site.register(Author)
+admin.site.register(BookInstance)
+admin.site.register(Genre)
+admin.site.register(Language)
```
Comment (faisalmaqsood01) [inline catalog/admin.py]: LGTM, just add EOL in this file.

Comment (hamadhassan3) [reply catalog/admin.py]: I think this file is from the first commit. This issue is already fixed in the files after the commit.

**Review by faisalmaqsood01 — APPROVED** [review]

File: catalog/models.py [hunk 2]
```diff
@@ -92,9 +92,20 @@ class Book(CatalogModel):
     genre = models.ManyToManyField(Genre, help_text='Select a genre for this book')
 
 
-    # The language is stored as a OneToOneField because a book can have only one language
+    # The language is stored as a Foreign Key because one language can be associated with many books
     # The Language class is already defined, so it can be used to define the relation
-    language = models.OneToOneField(Language, null = True, on_delete=SET_NULL)
+    language = models.ForeignKey(Language, null = True, on_delete=SET_NULL)
```
Comment (shahabejaz) [inline catalog/models.py]: You should not use space for arg value when passing to a function.

**Review by shahabejaz — COMMENTED** [review]
PR description is once again great! 👏 

Overall looks good to me, just one comment I have left. Before I give approval, you need to rebase this branch on top of main branch. Currently it is based on ModelsTutorial. Also you can safely delete ModelsTutorial branch because it is merged now.

File: catalog/models.py (hunk 2 above)
Comment (hamadhassan3) [reply catalog/models.py]: I have fixed this now.

**Review by shahabejaz — APPROVED** [review]

**Pull request merged**